deploy/kubernetes/csi-snapshotter/setup-csi-snapshotter.yaml

```diff
@@ -84,7 +84,7 @@ spec:
             - name: socket-dir
               mountPath: /csi
         - name: csi-snapshotter
-          image: gcr.io/k8s-staging-sig-storage/csi-snapshotter:v6.0.1
+          image: registry.k8s.io/k8s-staging-sig-storage/csi-snapshotter:v6.0.1
```

Comment:
Looks like this image doesn't exist? 

```
Error response from daemon: manifest for registry.k8s.io/k8s-staging-sig-storage/csi-snapshotter:v6.0.1 not found: manifest unknown: Failed to fetch "v6.0.1" from request "/v2/k8s-staging-sig-storage/csi-snapshotter/manifests/v6.0.1".
```

You need to replace `k8s-staging-sig-storage` with `sig-storage`

Reply:
Made the change